src/main/java/calculationEngine/entities/CeEntity.java

```diff
@@ -200,6 +201,16 @@ public void setHitPoints(int hitPoints) {
         this.hitPoints = hitPoints;
     }
 
+    public void dealDamage(int damage) {
+        this.hitPoints -= damage;
+        this.damages.add(damage);
+    }
+    public List<Integer> getDamages() {
+        List<Integer> tmp = new ArrayList<>(damages);
+        damages.clear();
```

Comment:
Why do you clear the list here. The functionality of this method is a littlebit strange to me.

Reply:
Otherwise it would just keep displaying all the damages that were dealt.